frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -458,6 +470,7 @@ function ReaderFooter:addToMainMenu(menu_items)
         table.insert(sub_items, getMinibarOption("frontlight"))
     end
     table.insert(sub_items, getMinibarOption("mem_usage"))
+    table.insert(sub_items, getMinibarOption("wifi_status"))
```

Comment:
Quoting from https://github.com/koreader/koreader/pull/3078#issuecomment-338476926

> Hi Guys, not sure if that's helpful but FYI in android we already have a function to check whether wifi is on or off
https://github.com/koreader/koreader/pull/3396/files#diff-c14ee0e93f7187eeb6f6fe8af52cf0b7R63
(it doesn't use ping)

Good point. That means that this can be merged as an Android-only feature. It might suffice to just stick an if isAndroid around this line but you'll have to test it.